<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CharSheetWindow</class>
 <widget class="QMainWindow" name="CharSheetWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#CharSheetWindow{
	background-color: rgb(0, 0, 90)
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="createCharFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>880</width>
      <height>550</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#createCharFrame{
	background-color: rgb(50, 50, 255)
}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="charImage">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>200</width>
       <height>200</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#charImage{
	background-color: rgb(27, 27, 27, 0.7);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="findFileButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>200</width>
       <height>200</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#findFileButton{
	background-color: rgb(0, 0, 0, 0)
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="charName">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>80</y>
       <width>200</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Fixedsys</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#charName{
	background-color: rgb(0, 0, 75, 0.2);
	border: 1px;
	border-radius: 15px;
	color: rgb(0, 0, 30)
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTableWidget" name="statsTable">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>170</width>
       <height>265</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <row>
      <property name="text">
       <string>Strength</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Agility</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Fighting</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Awareness</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Stamina</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Dexterity</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Intelect</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Presence</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Rank</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <item row="0" column="0">
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="1" column="0">
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="2" column="0">
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="3" column="0">
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="4" column="0">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="5" column="0">
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="6" column="0">
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="7" column="0">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
    </widget>
    <widget class="QTableWidget" name="statsTable_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>250</y>
       <width>165</width>
       <height>205</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <row>
      <property name="text">
       <string>Dodge</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Parry</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Will</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Fortitude</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Toughness</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Initiative</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Rank</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="1" column="0">
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="2" column="0">
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="3" column="0">
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="4" column="0">
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="5" column="0">
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
    </widget>
    <widget class="QListWidget" name="conditList">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>20</y>
       <width>141</width>
       <height>421</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#conditList{
	background-color: rgb(186, 186, 186);
}</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="resModSpinbox">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>50</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#resModSpinbox{
	background-color: rgb(186, 186, 186);
}</string>
     </property>
    </widget>
    <widget class="QComboBox" name="conditions">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>92</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#conditions{
	background-color: rgb(186, 186, 186);
}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addCondit">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>140</y>
       <width>220</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Fixedsys</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#addCondit{
	background-color: rgb(68, 68, 68);
	border: 1px ;
	border-radius: 20px
}

#addCondit:hover{
	background-color: rgb(42, 42, 42);
	border: 1px ;
	border-radius: 20px;
}</string>
     </property>
     <property name="text">
      <string>Add Condition</string>
     </property>
    </widget>
    <widget class="QPushButton" name="removeCondit">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>230</y>
       <width>220</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Fixedsys</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#removeCondit{
	background-color: rgb(68, 68, 68);
	border: 1px ;
	border-radius: 20px
}

#removeCondit:hover{
	background-color: rgb(42, 42, 42);
	border: 1px ;
	border-radius: 20px;
}</string>
     </property>
     <property name="text">
      <string>Remove Condition</string>
     </property>
    </widget>
    <widget class="QLabel" name="nameIndicator">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Fixedsys</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#nameIndicator{
	color: rgb(0, 0, 27)
}</string>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="resModIndicator">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>10</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Fixedsys</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#nameIndicator{
	color: rgb(0, 0, 27)
}</string>
     </property>
     <property name="text">
      <string>Resistance Modifier Conditions:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveButton">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>370</y>
       <width>220</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Fixedsys</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#saveButton{
	background-color: black;
	border: 1px ;
	border-radius: 20px;
	color: white
}

#saveButton:hover{
	background-color: white;
	border: 1px ;
	border-radius: 20px;
	color: black
}</string>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
